code/game/jobs/job/support.dm

```diff
@@ -1,117 +1,3 @@
-//Food
```

Comment:
Did you make any changes to this in the process of moving it?

Reply:
No changes to that part.
The only difference is the Quartermaster, Cargo Tech, and Shaft Miner jobs/outfits being moved to above the other service jobs, and their `selection_color` being changed.

Reply:
Gonna ask. What was the thought process behind moving this? It seems largely useless and just diff for the sake of it.

Reply:
![image](https://user-images.githubusercontent.com/15887760/140068156-e27643a2-6487-42de-9c0a-db2d479a2098.png)
Is the result if you don't move it (was curious myself if this would be the case)
So the view doesn't order it and instead uses the declaration order in the files

Reply:
^